docs/intro/from-ground-up.rst

```diff
@@ -181,8 +181,8 @@ can then refactor your ``BookPage`` class as follows:
     class BookPage(WebPage):
         def to_item(self) -> dict:
             return {
-                "url": self.response.url,
-                "title": self.response.css("h1").get(),
+                "url": self..url,
+                "title": self.css("h1").get(),
```

Comment:
..url is a typo.

I'm also not sure that the new example illustrates what's written before that: we explain that WebPage defines response argument, but then the example doesn't feature response.

Reply:
I’ll revert. I was thinking we mention helper methods but then don’t show them used, but showing response here is probably more important.

Reply:
https://github.com/scrapinghub/web-poet/pull/169/commits/28634d14ac80a46d64628a4a5777fdddcb35b029 ?